.travis.yml

```diff
@@ -0,0 +1,115 @@
+language: cpp
+
+matrix:
+  include:
+    - os: linux
```

Comment:
Can we also test on macOS?

Reply:
Yes, but I was thinking of adding that in a follow-on PR.